builds/e2e/e2e.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       arch: arm32v7
       artifactDir: iotedged-debian9-arm32v7
 
-    timeoutInMinutes: 90
+    timeoutInMinutes: 120
```

Comment:
Does this verification add ~30 mins?

Reply:
This extends the maximum time the tests can take on Pi. These changes don't really slow the tests down that much but in the worst case, where several tests fail, the Pi can easily go over the 90 min threshold. The agent can be left in a rather weird state if the test run times out--our cleanup code won't run for example--so absent some more complicated work on our part it's easiest to just prevent the timeout from happening.